asciidoc/sessions-and-transactions.adoc

```diff
@@ -263,16 +263,10 @@ When working with a Causal Cluster, transactions can be chained to ensure causal
 This means that for any two transactions, it is guaranteed that the second transaction will begin only after the first has been successfully committed.
 This is true even if the transactions are carried out on different physical cluster members.
 
-[NOTE]
---
-Due to protocol limitations, <<driver-transactions-auto-commit, auto-commit transactions>> cannot currently take part in the causal chain.
-While this is expected to change in a future protocol release, `session.run` calls should be avoided in places where causal consistency is important.
---
-
```

Comment:
@ali-ince Thank you!
Since there may be configurations that 1.7+ driver with pre-3.5 server I think we need to keep a note in there.
It could say something like:

[NOTE]	
--	
If you use the {driver-version} driver to connect to a Neo4j database that is version 3.4 or earlier, <<driver-transactions-auto-commit, auto-commit transactions>> cannot take part in the causal chain.	
In that configuration, `session.run` calls should be avoided in places where causal consistency is important.	
--	

It could possibly be moved to a less prominent place at the end of the section.

Reply:
Thanks @mariascharin. I've added the note just following the previous paragraph as I thought will be more visible.